ext/shave/extconf.rb

```diff
@@ -0,0 +1,16 @@
+require 'mkmf'
+require 'fileutils'
+
+$LOAD_PATH.push(__dir__ + '/../../lib')
+
+require 'datadog/ruby_core_source'
+
+dir = Datadog::RubyCoreSource.deduce_packaged_source_dir("ruby-#{RUBY_VERSION}-p#{RUBY_PATCHLEVEL}")
```

Comment:
PROBABLY we should move this whole bit of logic here in as well: https://github.com/DataDog/datadog-ruby_core_source/blob/4b3cc1ce88d628f291dbe4f8078aeb7ee708b173/lib/datadog/ruby_core_source.rb#L20-L24

Reply:
Also, maybe for 3.4 preview we might need to add stuff in here?

https://github.com/DataDog/datadog-ruby_core_source/blob/4b3cc1ce88d628f291dbe4f8078aeb7ee708b173/lib/datadog/ruby_core_source.rb#L6-L11